src/modules/graphics/graphics.c

```diff
@@ -1402,7 +1421,8 @@ Shader* lovrShaderCreate(ShaderInfo* info) {
   spv_result result;
   spv_info spv[2] = { 0 };
   for (uint32_t i = 0; i < stageCount; i++) {
-    result = spv_parse(info->stages[i]->data, info->stages[i]->size, &spv[i]);
+    lovrCheck(info->stages[i]->size < UINT32_MAX, "Shader too large");
+    result = spv_parse(info->stages[i]->data, (uint32_t)info->stages[i]->size, &spv[i]);
```

Comment:
I think it would be cleaner to change `spv_parse` to take a `size_t` instead of a `uint32_t` and remove the check/cast.